src/effects/audiounits/AudioUnitEffect.cpp

```diff
@@ -1043,34 +1043,54 @@ AudioUnitEffect::MakeInstance(EffectSettings &settings) const
 std::shared_ptr<EffectInstance>
 AudioUnitEffect::DoMakeInstance(EffectSettings &settings)
 {
-   OSStatus result;
-
    if (mMaster)
-      // Do common steps
-      InitializePlugin();
-
-   if (!mMaster) {
+      // This is a slave
```

Comment:
Not an object of this PR, but: I find the naming of this member `mMaster` very counterintuitive. The comment on its definition says `// non-NULL if a slave`, and the two lines here just confirm this.